OpenSim/Sandbox/ExampleHopperDevice/defineDeviceAndController.h

```diff
@@ -88,10 +88,11 @@ class Device : public ModelComponent {
     double getHeight(const SimTK::State& s) const {
         //TODO: Provide the name of the output corresponding to the hopper's
         //      height. You found this in Step 1, Task A.
-        const std::string hopperHeightOutput = "/Dennis/?????"; //fill this in
+        const std::string hopperHeightCoord = "/Dennis/?????"; //fill this in
```

Comment:
Please ensure all the comments have been updated where necessary. Making the comments agree with the code is critical for the example to be useful in a workshop setting. In this case, "Provide the name of the output" should be changed to "Provide the name of the coordinate".